plugins/env/test/samson_env/samson_plugin_test.rb

```diff
@@ -145,4 +150,53 @@ def call(user, action, group)
       assert call(users(:project_admin), :write, group)
     end
   end
+
+  describe 'view callbacks' do
```

Comment:
I'd not add tests for any of that ... maybe render the view with and without DEPLOYMENT_ENV_REPO to make sure it does not crash

also prefer not rendering views in isolation since it's kinda brittle, but render the whole action